docs/ai/conceptual/agents.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Let Agents and Copilots Bring Functionality to Your App"
```

Comment:
This title seems more task oriented than conceptual.

Reply:
You're right - I'll adjust it.